terraform/modules/appservice/main.tf

```diff
@@ -26,7 +26,7 @@ resource "azurerm_service_plan" "appserviceplan" {
 }
 
 resource "azurerm_linux_web_app" "appservice" {
-  name                       = "app-${var.resource_uid}"
+  name                       = "ase-${var.resource_uid}"
```

Comment:
`ase-` is for App Service Environment. For [Web Apps](https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations#compute-and-web) `app-` is used.

Reply:
Ah I didn't crtl+f `web app` in the list, makes sense!